packages/react/src/hooks/useOrganizations.ts

```diff
@@ -0,0 +1,31 @@
+import { CreateOrganizationParams, OrganizationResource } from '@clerk/types';
+import { useContext } from 'react';
+
+import { useClerk } from '../contexts';
+import { assertWrappedByClerkProvider } from '../contexts/assertHelpers';
+import { StructureContext } from '../contexts/StructureContext';
+
+type UseOrganizations = {
+  createOrganization: (
+    params: CreateOrganizationParams,
+  ) => Promise<OrganizationResource>;
+  getOrganizations: () => Promise<OrganizationResource[]>;
+  getOrganization: (
+    organizationId: string,
+  ) => Promise<OrganizationResource | undefined>;
+};
+
+export function useOrganizations(): UseOrganizations {
+  const structureCtx = useContext(StructureContext);
+  assertWrappedByClerkProvider(structureCtx);
+  const clerk = useClerk();
+
+  return {
+    createOrganization: clerk.createOrganization,
+    getOrganizations: clerk.getOrganizations,
+    getOrganization: async organizationId =>
+      await (
```

Comment:
@igneel64 Let's add this to ClerkJS. Don't forget that some users prefer the vanilla JS library.